BST Practice.py

```diff
@@ -0,0 +1,61 @@
+class Node(object):
+    def __init__(self, value):
+        self.value = value
+        self.left = None
+        self.right = None
+
+
+class BST(object):
+    def __init__(self, root):
+        self.root = Node(root)
+
+    def insert(self, new_val):
+        start = self.root
+        while start:
+            if start.value == new_val:
+                pass
+            elif start.value > new_val:
+                if start.left:
+                    start = start.left
+                else:
+                    start.left = new_val

```

Comment:
**start.left** should be of a **Node**, not an integer,  same as **start.right** below